lib/openstudio-standards/prototypes/common/objects/Prototype.hvac_systems.rb

```diff
@@ -2824,13 +2863,21 @@ def model_add_psz_vav(model,
       unitary_system.setCoolingCoil(clg_coil)
       unitary_system.setSupplementalHeatingCoil(supplemental_htg_coil)
       unitary_system.setName("#{zone.name} Unitary PSZ-VAV")
-      unitary_system.setString(2, 'SingleZoneVAV') # @todo add setControlType() method
+      # The following control strategy can lead to "Developer Error: Component sizing incomplete."
+      # EnergyPlus severe (not fatal) errors if there is no heating design load
+      unitary_system.setControlType('SingleZoneVAV')
+      unitary_system.maximumSupplyAirTemperature
```

Comment:
this line doesn't do anything

Reply:
@lymereJ : can we delete the line?
unitary_system.maximumSupplyAirTemperature